context/app/static/js/components/Detail/Citation/Citation.jsx

```diff
@@ -15,12 +16,25 @@ function buildNLMCitation({ contributors, title, timestamp }) {
   return `${joinedContributors}. ${title} [Internet]. HuBMAP Consortium; ${year}.`;
 }
 
-function Citation({ contributors, title, timestamp, url }) {
+function Citation({ contributors, title, timestamp, url, doi }) {
   const citation = buildNLMCitation({ contributors, title, timestamp });
 
   return (
     <SectionContainer id="citation">
-      <SectionHeader>Citation</SectionHeader>
+      <Flex>
+        <SectionHeader>Citation</SectionHeader>
+        <FlexRight>
+          <Button
+            color="primary"
+            variant="contained"
+            component="a"
+            target="_blank"
+            href={`https://search.datacite.org/works/${doi}`}
```

Comment:
We shouldn't need to encode the doi, right?

Reply:
I don't think so: If the service were structured differently, we might, but this works with a string like `10.35079/hbm876.xnrh.336` right in the path URL: We don't want to encode the `/` or anything like that.